app/controllers/searches_controller.rb

```diff
@@ -1,6 +1,20 @@
 class SearchesController < ApplicationController
-    def index
-        @places = SearchLogic.get_places_from_city_string(params[:city_name], params[:place_type])
-        render params[:place_type]
+  def index
+    if params[:city_name].present?
+      selected_place_types = []
+      selected_place_types << 'attractions' if params[:place_type_attractions] == 'attractions'
+      selected_place_types << 'hotels' if params[:place_type_hotels] == 'hotels'
+      selected_place_types << 'restaurants' if params[:place_type_restaurants] == 'restaurants'
```

Comment:
Not necessary for this PR, but one alternative (not necessarily better) approach is to use an array in the form such that the URL ends up being like `&place_types[]=attractions&place_types[]=hotels&place_types[]=restaurants`. Then you can set `selected_place_types` in one line and only have to validate that it's an array:

```ruby
selected_place_types = params[:place_types]
```

However, you'd need to validate what the user passes in, which could get messy, so don't worry about it.